### DISCUSSION TITLE
Difference between using Postgres and TimescaleDB driver

### DISCUSSION THREAD

**Discussion by hobbes1069** [dbeaver/dbeaver#20758] [Q&A] [answered]
I can connect to my database using either the Postgres driver or the TimescaleDB driver but I can't find any documentation on what difference it makes.

What can I do using the TimescaleDB driver that I can't do with the Postgres driver?
How does DBeaver treat the database differently?

**Comment by bobbyiliev** [comment, marked as answer]
The two drivers are more or less the same, the TimescaleDB driver just extends the Postgres Driver so that if people were not aware that they could use the Postgres driver they would find the TimescaleDB driver easier.

You can check what the TimescaleDB driver overrides here:

> https://github.com/dbeaver/dbeaver/blob/devel/plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/impls/PostgreServerTimescale.java

**Comment by hobbes1069** [comment]
Thanks!
